src/spikeinterface/benchmark/benchmark_base.py

```diff
@@ -305,6 +392,13 @@ def get_sorting_analyzer(self, case_key=None, dataset_key=None):
         # sorting_analyzer = load_sorting_analyzer(folder)
         # return sorting_analyzer
 
+    def compute_gt_unit_locations(self, dataset_keys=None, **method_kwargs):
+        if dataset_keys is None:
+            dataset_keys = list(self.datasets.keys())
+        for dataset_key in dataset_keys:
+            sorting_analyzer = self.get_sorting_analyzer(dataset_key=dataset_key)
+            sorting_analyzer.compute("unit_locations", **method_kwargs)
+
```

Comment:
Why a special focus on gt_unit_locations? I mean such a function could be great, to compute quantities on the main analyzer, but maybe it could be more generic and allow any extension to be computed? Or otherwise, let's discuss why only gt_unit_locations

Reply:
This is because the the `plot_performance_vs_depth_and_snr` uses it

Reply:
Changed in last commit to have a more general `compute_analyzer_extension` and `get_gt_unit_locations` (which uses the `gt_unit_location` property if available)